math/utils.go

```diff
@@ -567,3 +567,46 @@ func GetQuantileOfDecs(
 	}
 	return ret, nil
 }
+
+// MedianAbsoluteDeviation calculates the median absolute deviation of a slice of Dec values.
+// Returns (MAD, median, error)
+func MedianAbsoluteDeviation(data []Dec) (Dec, Dec, error) {
```

Comment:
totally a nitpick so feel free to ignore if lazy but you could name the variables rather than having to have the comment
```suggestion
// MedianAbsoluteDeviation calculates the median absolute deviation of a slice of Dec values.
func MedianAbsoluteDeviation(data []Dec) (medianAbsoluteDeviation Dec, median Dec, err error) {
```

Reply:
good catch, added alongside more stuff so discarding your suggestion :heart: